paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md

```diff
@@ -1,9 +1,9 @@
 # INT8 MKL-DNN quantization 
 
-This document describes how to use Paddle inference Engine to convert the FP32 model to INT8 model on ResNet-50 and MobileNet-V1. We provide the instructions on enabling INT8 MKL-DNN quantization in Paddle inference and show the ResNet-50 and MobileNet-V1 results in accuracy and performance.
+This document describes how to use Paddle inference Engine to convert the FP32 model to INT8 model on following 7 models: ResNet-50, ResNet-101, MobileNet-V1, MobileNet-V2, VGG16, VGG19 and GoogleNet. We provide the instructions on enabling INT8 MKL-DNN quantization in Paddle inference and show all the 7 models' results in accuracy and performance.
```

Comment:
`and show all the 7 models' results in accuracy and performance` -> `and show the accuracy and performance results of all 7 models`.

Reply:
Done